### DISCUSSION TITLE
[iOS Pag]Pag播放时进行UIView animation会导致Pag播放卡顿

### DISCUSSION THREAD

**Discussion by albertlianglt** [Tencent/libpag#1934] [General] [answered]
在使用过程中遇到了Pag播放时进行UIView animation会导致Pag播放卡顿，请问有没有使用或者优化建议？setSync也是采用的默认为NO的情况。

**Comment by domchen** [comment, marked as answer]
1. 采用 PAGImageView 替代 PAGView  2. 优化素材的性能，参考https://pag.art/docs/optimization.html
